index.js

```diff
@@ -68,11 +73,15 @@ let definitionCommand = require('./commands/definition')();
 let goodMorningGreetingCommand = greetingCommand(channelIdForGeneral, iconUrl, bot);
 let goodNightGreetingCommand = greetingCommand(channelIdForGeneral, iconUrl, bot, `Good night ${appName}`);
 let getAstronomyPictureOfTheDayCommand = require('./commands/get_astronomy_picture_of_the_day')(channelIdForRandom, iconUrl, bot);
+let announceEventsHappeningTodayCommand = require('./commands/announce_events_happening_today')(channelIdForEvents, iconUrl, bot);
+let announceEventsHappeningTomorrowCommand = require('./commands/announce_events_happening_tomorrow')(channelIdForEvents, iconUrl, bot);
 
 // scheduled messages
 schedule.scheduleJob('30 9 * * *', goodMorningGreetingCommand);
```

Comment:
I think we should rename `schedule` to `scheduler`. It makes it read better:

`scheduler.scheduleJob('30 23 * * *', goodNightGreetingCommand);`